src/app/components/CourseChoiceChosen/CourseChoiceChosen.module.css

```diff
@@ -0,0 +1,11 @@
+.courseChoiceChosen {
+  display: grid;
+  grid-template-columns: 5fr 1fr;
+  align-items: center;
+  column-gap: 0.5em;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Maybe again: `margin: 0 auto` ?

Reply:
It can be improved to margin: auto. Thank you!